src/solana/solanaInfo.ts

```diff
@@ -6,6 +6,14 @@ import type { SolanaTools } from './SolanaTools'
 import type { SolanaNetworkInfo } from './solanaTypes'
 
 const builtinTokens: EdgeTokenMap = {
+  DezXAZ8z7PnrnRJjz3wXBoRgixCa6xjnB7YaB1pPB263: {
```

Comment:
Are Solana tokenId's not lower case? I know on Ethereum we forced these to be lowercase, but maybe this chain does it differently.

Reply:
Didn't bother to lowercase it when originally implemented. Other chains outside of evms are the same